dwave/preprocessing/presolve/cypresolve.pyx

```diff
@@ -127,6 +124,57 @@ cdef class cyPresolver:
         """Load the default presolvers."""
         self.cpppresolver.load_default_presolvers()
 
+    cpdef bint normalize(self) except*:
+        """Apply any loaded presolve techniques to the held constrained quadratic model.
```

Comment:
Edit first line of docstring as it may be copied from the method `presolve()` docstring